exercises/starknet/README.md

```diff
@@ -1,7 +1,87 @@
 # Starknet Smart Contracts
 
-Cairo is the language used to develop Smart Contracts on Starknet.
+A Starknet contract is pretty much a Cairo module, or rather, it's represented by one.
+
+Starknet contract don't require a `main` function.
```

Comment:
Once again I don't really understand why this is part exists - it should be in the book directly

Reply:
Some basic info, to get started with...